lib/components/SDescFile.vue

```diff
@@ -0,0 +1,98 @@
+<script setup lang="ts">
+import IconDownloadSimple from '@iconify-icons/ph/download-simple-bold'
+import IconFileText from '@iconify-icons/ph/file-text-bold'
+import { computed } from 'vue'
+import { isArray } from '../support/Utils'
+import SButton from './SButton.vue'
+import SDescEmpty from './SDescEmpty.vue'
+import SIcon from './SIcon.vue'
+
+export interface File {
```

Comment:
we probably should rename this to something else. `File` is there in lib dom too. It might be confusing.

Reply:
Hmmm good point. Maybe we'll just call it `Item` ? 🤔

Reply:
Yeah that or something scoped like `DescFile`/`FileType`. Anything would work I guess.

Reply:
Gotcha, I'm gonna go with Item I guess 👍

Reply:
Updated 👍